src/Domain/DependencyInjection.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Reflection;
+using FluentValidation.AspNetCore;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace CleanCompanyName.DDDMicroservice.Domain;
+
+public static class DependencyInjection
+{
+    public static IServiceCollection AddDomain(this IServiceCollection services)
+    {
+        services.AddFluentValidation(config =>
+        {
+            config.RegisterValidatorsFromAssembly(
+                Assembly.GetExecutingAssembly(),
+                includeInternalTypes: true);
+            });
```

Comment:
```suggestion
        services.AddFluentValidation(config =>
        {
            config.RegisterValidatorsFromAssembly(
                Assembly.GetExecutingAssembly(),
                includeInternalTypes: true);
        });
```
Btw, what about configuring resolvers here instead of using those `.Global` static members?